search-enterprise/latest/support/index.md

```diff
@@ -11,7 +11,8 @@ The following are the supported platforms for Search Enterprise 3.1.x:
 | Content Services 7.2.x | Including Alfresco ActiveMQ 5.15.8, Alfresco Transform Service 1.4.1 and PostgreSQL 11.7 |
 | Content Services 7.1.x | Including Alfresco ActiveMQ 5.15.8, Alfresco Transform Service 1.4.1 and PostgreSQL 11.7 |
 | JDK 11 or OpenJDK 11 | |
-| Elasticsearch server 7.10.1 | |
-| Alfresco Elasticsearch Connector 3.1.x | |
+| Elasticsearch server 7.17.3 | |
+| Opensearch server 1.3.2 | |
+| Alfresco Elasticsearch Connector 3.2.x | |
 
-> **Note:** The Elasticsearch server does not require any additional software from Alfresco in order to be used by Alfresco Search Enterprise 3.1
+> **Note:** The Elasticsearch/Opensearch server does not require any additional software from Alfresco in order to be used by Alfresco Search Enterprise 3.2
```

Comment:
Slightly clunky. I would change to "Elasticsearch/Opensearch does not require any additional software from Alfresco in order to be used by Alfresco Search Enterprise 3.2"

Reply:
Changed